### ISSUE TITLE
[Bug]: notifications should be synced across open browser tabs

### ISSUE THREAD

**Issue by alexanderdd** [nextcloud/server#49921]
### ⚠️ This issue respects the following points: ⚠️

- [x] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [x] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [x] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [x] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

.

### Steps to reproduce

1. have multiple tabs with NC open
2. receive notifications
3. dismiss notifications in one tab
4. go to different tab
5. the dismissed notifications are still there

### Expected behavior

- notifications should be synced across open browser tabs
- dismissed notifications should be dismissed in all browser tabs

### Nextcloud Server version

30

### Operating system

None

### PHP engine version

None

### Web server

None

### Database engine version

None

### Is this bug present after an update or on a fresh install?

None

### Are you using the Nextcloud Server Encryption module?

None

### What user-backends are you using?

- [ ] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```json

```

### List of activated Apps

```shell

```

### Nextcloud Signing status

```shell

```

### Nextcloud Logs

```json

```

### Additional info

it is very common that people have NC open in multiple tabs/windows

it is confusing that a user action in one tab is not respected in another tab

**Comment by alexanderdd** [comment]
@szaimen can you triage this?

**Comment by szaimen** [comment]
Yes, moving to the notifications app as I think it needs to be handled there
